### PR TITLE
Sdk feature store ver1

### PR DESCRIPTION
**Pull request by SamyuktaDR**
If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [x] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

If you are opening a PR for `Community Notebooks` under the [notebooks/community](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/community) folder:
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/community/CODEOWNERS) file under the `Community Notebooks` section, pointing to the author or the author's team.
- [ ] Passes all the required formatting and linting checks. You can locally test with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).

If you are opening a PR for `Community Content` under the [community-content](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/community-content) folder:
- [ ] Make sure your main `Content Directory Name` is descriptive, informative, and includes some of the key products and attributes of your content, so that it is differentiable from other content
- [ ] The main content directory has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/community-content/CODEOWNERS) file under the `Community Content` section, pointing to the author or the author's team.
- [ ] Passes all the required formatting and linting checks. You can locally test with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).

### CODE REVIEW

File: notebooks/official/feature_store/sdk-feature-store.ipynb [hunk 1]
```diff
@@ -262,7 +262,15 @@
       },
```
Comment (andrewferlitsch) [inline notebooks/official/feature_store/sdk-feature-store.ipynb:262]: <p>Missing H1 title heading</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/790/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (andrewferlitsch) [inline notebooks/official/feature_store/sdk-feature-store.ipynb:262]: <p>Split into 4 separate cells.</p><p>Move Dataset after Objective </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/790/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (andrewferlitsch) [inline notebooks/official/feature_store/sdk-feature-store.ipynb:262]: <p>combine these two cells</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/790/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (andrewferlitsch) [inline notebooks/official/feature_store/sdk-feature-store.ipynb:262]: <p>combine into one cells</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/790/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Comment (andrewferlitsch) [inline notebooks/official/feature_store/sdk-feature-store.ipynb:262]: <p>replace with SDK call</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/790/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Comment (SamyuktaDR) [reply notebooks/official/feature_store/sdk-feature-store.ipynb:262]: <p>Most of the featurestore services are in beta or older versions (v1, v1beta1). Can we use them?</p>

Comment (andrewferlitsch) [reply notebooks/official/feature_store/sdk-feature-store.ipynb:262]: I would recommend the SDK versions from this notebook:
https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/community/ml_ops/stage2/get_started_vertex_feature_store.ipynb

**Review by andrewferlitsch — APPROVED** [review]

**Comment by andrewferlitsch** [comment]
testing failed:

Input In [13], in <cell line: 1>()
Step #6: ----> 1 fs = Featurestore(
Step #6:       2     featurestore_name=FEATURESTORE_ID,
Step #6:       3     project=PROJECT_ID,
Step #6:       4     location=REGION,
Step #6:       5 )
Step #6:       6 print(fs.gca_resource)
Step #6: 
...
Step #6: File /workspace/workspace/env/lib/python3.10/site-packages/google/api_core/grpc_helpers.py:52, in _wrap_unary_errors.<locals>.error_remapped_callable(*args, **kwargs)
Step #6:      50     return callable_(*args, **kwargs)
Step #6:      51 except grpc.RpcError as exc:
Step #6: ---> 52     raise exceptions.from_grpc_error(exc) from exc
Step #6: 
Step #6: NotFound: 404 The Featurestore does not exist.

**Comment by SamyuktaDR** [comment]
All checks passed. Ready to be merged.

**Pull request merged**